config/webpack/webpack.config.js

```diff
@@ -0,0 +1,15 @@
+const { env, webpackConfig } = require('@rails/webpacker')
```

Comment:
Adding this file is the only change required for the migration to using the simpler, single config file.